wallet/res/layout/unable_to_connect_view.xml

```diff
@@ -1,72 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<merge xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I guess with the new method of showing blockchain sync "Syncing 30%" on the home screen, we may no longer have a means of telling the user that the device is offline.

Reply:
I mean we still have this red panel on the home screen when the device is offline, no? This particular layout wasn't used anywhere.